### ISSUE TITLE
Congratulations and a couple of queries regarding the data

### ISSUE THREAD

**Issue by mray0n** [snives/F1-Media-Coverage-Analysis#1]
Hi @snives 
Just to tell you that it is a very interesting work this that you have done
Just a couple of questions regarding the bot to scrap the web. Can you share the principles of the bot? I mean, you look up for an rss feed, and depending on the words decide whether the article was F1 related or not? Or you directy search for specialised rss feed of F1, and then work with the data?
And regarding the processing of the data, did you use python or r or any other tool? Thanks!

**Comment by snives** [comment]
Thanks!

You are correct, the bot just reads RSS feeds from F1 websites.  As for
processing it identifies those articles which mentioned the specific
driver.  It utilized a relational database to perform the query.

Do you have a need or interest in doing something similar?

-Snives

**Comment by mray0n** [comment]
No, I was just curious. I am learning Natural Language processing and being F1 my passion was wondering around to see what interesting things could be out there, just that.
Thanks!

**Comment by snives** [comment]
Yeah that's cool, obviously I'm a huge fan as well. I would like to extend
this to gain further insights into F1, such as identifying trending topics
and not just following drivers.

Are you taking a class on NLP?  Which one?  Is it good?
